docs/docs/docs/developer-resources/testing.md

```
@@ -66,6 +66,27 @@ The GraphQL endpoint for handling `subscriptions` is:
+1. Launch the terminal application on your device.
+
+2. **Retrieve IPv4 Address**:
+    - **For Windows Users**: Enter the command `ipconfig`.
+    - **For Linux/OSX Users**: Enter the command `ifconfig`.
+    - Copy the `IPv4 Address` displayed (e.g., `192.168.12.233`).
+
+3. Make sure both your mobile and your computer are connected to the same network.
+
+4. Use the following format for your endpoint:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)